onbuild/Dockerfile

```diff
@@ -0,0 +1,29 @@
+#
+# Dockerfile for Storm Mesos framework
+#
+FROM mesosphere/mesos:0.27.0-0.2.190.ubuntu1404
```

Comment:
Yup, we do *want* to do that!   Unfortunately, the mesos packages published by Mesosphere do not have a predictable name for each version (note the "-0.2.190" portion above, it is present in the published rpm/debs as well, and is an artifact from the build process used by Mesosphere).  I've raised that issue with Mesosphere already and they are supposedly planning on getting rid of these extra obfuscating bits.